content/2020/2020-09/2020-09-25-pulsar-weekly.md

```diff
@@ -0,0 +1,133 @@
+date: 2020-09-19 ~ 2020-09-25
+description: "This is the Pulsar community weekly update for 2020-09-19 ~ 2020-09-25, with updates on Pulsar client, broker, Pulsar SQL, Pulsar Functions, and so on."
+id: "2020-09-25-pulsar-weekly"
+---
+
+## [Pulsar Community Weekly Update] 2020-09-19 ~ 2020-09-25
+
+This is the Pulsar community weekly update for 2020-09-19 ~ 2020-09-25, with updates on Pulsar client, broker, Pulsar SQL, Pulsar Functions, and so on.
+
+### Development
+
+- [Broker] Use SNI for the TLS-enabled Pulsar Java broker client.
+
+    https://github.com/apache/pulsar/pull/8117(@[racorn](https://github.com/racorn))
+
+- [Pulsar SQL] Replace `com.ning.asynchttpclient` with `org.asynchttpclient`.
+
+    https://github.com/apache/pulsar/pull/8099(@[zymap](https://github.com/zymap))
+
+- [Functions] Support specifying the secrets class in the localrunner builder.
+
+    https://github.com/apache/pulsar/pull/8127(@[srkukarni](https://github.com/srkukarni))
+
+- [Websocket] The WebSocket API supports unsubscribing from a topic.
+
+    https://github.com/apache/pulsar/pull/8068(@[315157973](https://github.com/315157973))
+
+- [Websocket] The WebSocket interface supports decoding URL for the topic name and subscription.
+
+    https://github.com/apache/pulsar/pull/8072(@[315157973](https://github.com/315157973))
+
+- [Broker] Support configuring the number of channels per bookie.
+
+    https://github.com/apache/pulsar/pull/7910(@[rdhabalia](https://github.com/rdhabalia))
+
+- [Functions] Support specifying the environment based on the secrets provider for source/sink/functions localrunner.
+
+    https://github.com/apache/pulsar/pull/8098(@[srkukarni](https://github.com/srkukarni))
+
+- [Client] Add a REST endpoint to trigger `backlogQuotaCheck`.
+
+    https://github.com/apache/pulsar/pull/8045(@[315157973](https://github.com/315157973))
+
+- [pulsar-client] Support non-durable subscription for pulsar-client commands.
+
+    https://github.com/apache/pulsar/pull/8100(@[315157973](https://github.com/315157973))
+
+- [Functions] Enable reporting functions worker JVM metrics through Prometheus.
+
+    https://github.com/apache/pulsar/pull/8097(@[jerrypeng](https://github.com/jerrypeng))
+
+- [Broker] Add replicated subscription check to `checkInactiveSubscriptions`.
+
+    https://github.com/apache/pulsar/pull/8066(@[shustsud](https://github.com/shustsud))
+
+- [Client] Support forcedly deleting all resources related to the namespace.
+
+    https://github.com/apache/pulsar/pull/7993(@[murong00](https://github.com/murong00))
+
+- [Topic Policy] Support setting, getting, and removing the subscription dispatch rate on the topic level.
+
+    https://github.com/apache/pulsar/pull/8087(@[hangc0276](https://github.com/hangc0276))
+
+- [Topic Policy] Support setting, getting, and removing the subscribe rate on the topic level.
+
+    https://github.com/apache/pulsar/pull/7991(@[hangc0276](https://github.com/hangc0276))
```

Comment:
duplicated?

Reply:
no, these are two kinds of setting, one is for dispatch rate and the other is the subscription rate